src/bots/telegram/client.js

```diff
@@ -108,30 +123,62 @@ class ForwardedMessage extends Message {
   }
 
   toString(): string {
-    const msg = this.msg;
+    const { msg } = this;
 
     if (!msg.forward_from) {
       return '';
     }
 
     const nick = Name.from(msg.forward_from);
 
-    // adds quoting brackets
-    const message = msg.text ? msg.text.replace(/\n/g, '\n>> ') : '';
+    delete msg.forward_from;
```

Comment:
That looks quite strange and dangerous. `toString` method deleting properties from `this.msg`? Nonsense!